openstudio-gems.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   # gem version is specified in gemspec, gem source/location (e.g. github branch) can be specified in Gemfile
   # runtime dependency versions can be loosened while in development on branches if needed
   # runtime dependency versions should be specified as exact versions when merged to master or develop
-  spec.add_dependency 'openstudio-extension', '0.6.0'
+  spec.add_dependency 'openstudio-extension', '0.6.1'
```

Comment:
Just an FYI, that the dependencies should probably be carefully adjusted in conjunctioin with the various repos: 
https://github.com/NREL/openstudio-extension-gem/blob/3adb1c13636245a1fe7a51c732b09249c67b24cf/openstudio-extension.gemspec#L35

Reply:
I believe the openstudio-extension has been updated to 0.7.0 and openstudio-standards has updated to 0.5.0.rc1. Those are the things I need update. Is it something else?